### ISSUE TITLE
Allow users to magnify beyond 200% and maintain function of the app (PDFs)

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/support#85]
This issue is part of our effort to become WCAG 2.0 AA compliant. 

1.4.4: Resize text
- [ ] Allow users to magnify beyond 200% and maintain function of the app (A)

Problem:
Users should be able to increase the size of text to 200% without loss of content or
functionality and without requiring horizontal scrolling. This works well for HTML documents, but doesn't work for PDFs. 

Steps to reproduce:
- Activate Hypothesis over a PDF - [This one, for example](https://d242fdlp0qlcia.cloudfront.net/uploads/2018/05/25160432/HypothesisWCAGReview20180514.pdf).
- Using your keyboard, attempt to magnify your browser tab by 200%

Expected results:
Like with an HTML page, both the document and sidebar are magnified

Actual results:
The PDF content is magnified, but the sidebar is not:

<img width="1604" alt="Screen Shot 2019-12-17 at 1 10 20 PM" src="https://user-images.githubusercontent.com/12087705/71022544-b594c880-20ce-11ea-9fbb-ae6e771774ef.png">

Compare this to the HTML experience:
<img width="1605" alt="Screen Shot 2019-12-17 at 1 10 52 PM" src="https://user-images.githubusercontent.com/12087705/71022557-bd546d00-20ce-11ea-95f5-e1ac73c18949.png">

Notes:
See W3C guidelines for scale/magnification here: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html
